QuickOSM/test/test_preset_parser.py

```diff
@@ -0,0 +1,185 @@
+import collections as col
+import unittest
+import xml.dom.minidom as xml
+
+from QuickOSM.core.parser.preset_parser import PresetsParser
+
+__copyright__ = 'Copyright 2021, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+class TestQuickOSMWidget(unittest.TestCase):
+
+    def setUp(self):
+        self.maxDiff = None
+
+    def test_parser(self):
+        """ Test if the parser return the format expected"""
+
+        parser = PresetsParser()
+        data = col.namedtuple('data', ['type', 'ancestors', 'heirs'])
+
+        # Simple file 1 group 1 item
```

Comment:
You can split in different `def`. it's a better practice.
So test 1 can fail, without blocking `test 2` for instance.